extensions/vscode/syntaxes/wake.tmLanguage.json

```diff
@@ -152,4 +152,4 @@
 			]
 		}
 	}
-}
\ No newline at end of file
+}
```

Comment:
can't remember what the policy on trailing newlines is, but looks like it was added here when it was previously missing

Reply:
non-"binary"-data files like this one should have the last character of the file be a newline